src/Windows/dcap_provider.sln

```diff
@@ -7,32 +7,38 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "dcap_provider", "dll\dcap_p
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "dcap_provider_tests", "dcap_provider_tests\dcap_provider_tests.vcxproj", "{7F43D2BD-351E-47EF-8E65-91337330AE89}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "dcap_provider_ut", "UnitTest\dcap_provider_ut.vcxproj", "{C356FCD3-13E4-40F1-9FDD-D2E68FA97885}"
-EndProject
-Project("{930C7802-8A8C-48F9-8165-68863BCCD9DD}") = "AzureDCAPSetupProject", "AzureDCAPSetupProject\AzureDCAPSetupProject.wixproj", "{A5E8BFB6-3DBE-498F-9431-544D7C09B1F0}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "UnitTests", "UnitTests\UnitTests.vcxproj", "{102D922A-E4D5-4B3F-9F83-BB001ACEFCBF}"
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86

```

Comment:
Can you remove the x86 configuration? We don't build and deploy x86 do we?

Reply:
Yes, I will remove it.